floor/floor/util/color_utils.py

```diff
@@ -121,6 +121,13 @@ def hex_to_rgb(value):
     return tuple(int(value[i:i + lv // 3], 16) for i in range(0, lv, lv // 3))
 
 
+def blend_pixel_nonalpha(bottom, top):
```

Comment:
Is the idea to have a bunch of `blend_pixel_` methods?  For `nonalpha`, maybe `copy` instead to describe what's happening rather than a quality of the pixels involved?  Or maybe even `blend_mode_copy`?

I was trying to find some standard mixing names and found this:

http://illu.tistory.com/1254

It seems to assume the `float a, float b` values are [0.0, 1.0] which works out.  These would probably be a good TODO for adding later.

Reply:
+1 to `copy` over blend! I didn't have other methods in mind but wanted some sort of descriptive name to say "yo this just treats black as alpha 1.0", like you suggestion better.